### PR TITLE
Speaker notes for methods.md

### PR DESCRIPTION
**Pull request by cchiw**

### CODE REVIEW

File: src/methods.md [hunk 1]
```diff
@@ -25,3 +25,17 @@ fn main() {
 }
 ```
 
+<details>
+
+Key Points:
+* It can be helpful to introduce methods by comparing them to functions.
+  * Methods are called on an instance of a struct, the first parameter represents the instance as `self`.
```
Comment (qwandor) [inline src/methods.md:32]: Not necessarily a struct, they can be implemented for any type (e.g. an enum or a primitive).

Comment (cchiw) [reply src/methods.md]: updated, what do you think

Comment (qwandor) [reply src/methods.md]: I've reworded it in a way that I think is clearer.

Comment (cchiw) [reply src/methods.md]: LGTM

**Comment by cchiw** [comment]
Add Issues #274

**Pull request merged**